sections/Product/FeaturedGallery.tsx

```diff
@@ -0,0 +1,58 @@
+import type { ImageWidget } from "apps/admin/widgets.ts";
+import Image from "apps/website/components/Image.tsx";
+import type { Product } from "apps/commerce/types.ts";
+import ProductCard from "../../components/product/ProductCard.tsx";
+export interface Props {
+    title: string;
+    idProduct: Product[] | null;
```

Comment:
Melhor usar apenas `product` para evitar erros de entendimento em manutençoes no futuro pois nao é um ID (string/number), e sim um objeto
```suggestion
    product: Product[] | null;
```